src/Components/Signup.jsx

```diff
@@ -17,6 +18,7 @@ class Signup extends Component {
     name: '',
     city: '',
     country: 'Sweden',
+    role: '',
```

Comment:
where do u set the role?

Reply:
```
handleSignup = () => {
    const { registerUser } = this.props;
    const { email, name, nickname, password, password_confirmation, city, country, role } = this.state;
    registerUser({ email, name, nickname, password, password_confirmation, city, country, role })
      .then(
        console.log('yiihaaaa')
      )
      .catch(error => {
        this.setState({errorMessage: error.response.data.errors.full_messages[0]}) 
      })
  }
```

Reply:
:)

Reply:
the role is empty in the constructor, where do u set the role to 'user'?

Reply:
well, we pass in the props from registerUser attributes in reduxTokenAuthConfig and set the state of the user to whatever we have the user set to in the backend, however, maybe we dont need it in the signup and only in login?